managed-databases/postgresql-and-mysql/how-to/enable-autobackup.mdx

```diff
@@ -3,44 +3,50 @@ meta:
   title: How to enable autobackup for Databases
   description: This page explains how to enable autobackup for Databases
 content:
-  h1: How to enable autobackup for Databases
+  h1: How to manage autobackups for Database Instances
   paragraph: This page explains how to enable autobackup for Databases
-tags: managed-dabatase autobackup database 
+tags: managed-dabatase autobackup database
 dates:
-  validation: 2023-01-19
+  validation: 2023-04-18
   posted: 2019-05-11
 categories:
   - managed-databases
   - postgresql-and-mysql
 ---
 
-Backups are logical data dumps of an individual database. They are useful when you need to restore a specific logical database. 
+With the autobackup feature the creation and management of your Database Instance backups is automated according to your backup schedule configuration.
 
-For Database Instances that use Block Storage volumes, Managed Database for PostgreSQL and MySQL uses the [Snaphots](/managed-databases/postgresql-and-mysql/how-to/manage-snapshots/) feature to create autobackups. When the autobackup feature is enabled, Managed Database for PostgreSQL and MySQL create and manage backups according to your backup schedule configuration. 
+Two types of autobackups are available at Scaleway: [backups](/managed-databases/postgresql-and-mysql/concepts/#database-backup) and [snapshots](/managed-databases/postgresql-and-mysql/concepts/#database-snapshot).
+
+Backups are used when your Database Instance's storage option is a local volume. Snapshots are available for Database Instances that use [Block Storage volumes.
```

Comment:
```suggestion
Backups are used when your Database Instance's storage option is a local volume. Snapshots are available for Database Instances that use Block Storage volumes.
```

Reply:
We might add that backups are also possible on block storage for volumes that do not exceed 585GB